packages/headless/src/controllers/recent-queries-list/headless-recent-queries-list.ts

```diff
@@ -167,91 +37,23 @@ export function buildRecentQueriesList(
   engine: SearchEngine,
   props?: RecentQueriesListProps
 ): RecentQueriesList {
-  if (!loadRecentQueriesListReducer(engine)) {
-    throw loadReducerError;
-  }
-
-  const controller = buildController(engine);
+  const coreController = buildCoreRecentQueriesList(engine, props);
```

Comment:
```suggestion
  const coreRecentQueriesListController = buildCoreRecentQueriesList(engine, props);
```

Just a suggestion, I think it would be more descriptive, I leave it to you if you think the name is too long